distribution/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.alfresco</groupId>
+        <artifactId>alfresco-hxinsight-connector-parent</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <name>Alfresco HxInsight Connector :: Distribution zip</name>
+    <artifactId>alfresco-hxinsight-connector-distribution</artifactId>
+    <packaging>jar</packaging>
+
+    <profiles>
+        <profile>
+            <id>distribution</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
This is intentional - zip should be created every time app will be built?

Reply:
Isn't the distribution module also built every time we use `mvn package` in root folder?
That was my first assumption, that zip file will be created if particular profile will be passed through `-P`

Reply:
I guess it boils down to expectations about what should happen if you build in the root folder. I would normally expect everything to build by default, but I'm aware this doesn't happen in many projects.

I don't mind adding profiles to support different use cases, but maybe we'll know better what we want after working on the project for a while? I'm happy if you want to make a PR for it or raise a ticket to do something about this?

Reply:
I think that we should build distribution `zip` only when it's required, similarly to building Docker images in other projects. This could save us few seconds while building app localy.

Reply:
I'm OK with this approach to work a bit on the project and see later if this build of `zip` distribution will give us some gain